### DISCUSSION TITLE
Doris 1.2.6 BE failed to start

### DISCUSSION THREAD

**Discussion by SamuelBin** [apache/doris#29975] [A - General / Q&A]
start time: Fri Jan 12 17:45:54 CST 2024
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/data/doris/be/lib/hadoop_hdfs/common/lib/slf4j-reload4j-1.7.36.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/data/doris/be/lib/java-udf-jar-with-dependencies.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Reload4jLoggerFactory]
terminate called after throwing an instance of 'std::system_error'
  what():  Resource temporarily unavailable
*** Query id: 0-0 ***
*** Aborted at 1705052755 (unix time) try "date -d @1705052755" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGABRT unkown detail explain (@0x3e80000772f) received by PID 30511 (TID 0x7f0cbfb97700) from PID 30511; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# 0x00007F0E033BD280 in /lib64/libc.so.6
 2# gsignal in /lib64/libc.so.6
 3# abort in /lib64/libc.so.6
 4# __gnu_cxx::__verbose_terminate_handler() [clone .cold] at ../../../../libstdc++-v3/libsupc++/vterminate.cc:75
 5# __cxxabiv1::__terminate(void (*)()) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
 6# 0x000055E8514BADC1 in /data/doris/be/lib/doris_be
 7# 0x000055E8514BAF14 in /data/doris/be/lib/doris_be
 8# std::__throw_system_error(int) at ../../../../../libstdc++-v3/src/c++11/system_error.cc:338
 9# 0x000055E85153D03D in /data/doris/be/lib/doris_be
10# rocksdb::ThreadPoolImpl::Impl::StartBGThreads() in /data/doris/be/lib/doris_be
11# rocksdb::ThreadPoolImpl::Impl::SetBackgroundThreadsInternal(int, bool) in /data/doris/be/lib/doris_be
12# rocksdb::DBOptions::IncreaseParallelism(int) in /data/doris/be/lib/doris_be
13# doris::OlapMeta::init() at /root/doris/be/src/olap/olap_meta.cpp:68
14# doris::DataDir::_init_meta() at /root/doris/be/src/olap/data_dir.cpp:202
15# doris::DataDir::init() at /root/doris/be/src/olap/data_dir.cpp:115
16# std::thread::_State_impl<std::thread::_Invoker<std::tuple<doris::StorageEngine::_init_store_map()::{lambda()#1}> > >::_M_run() at /var/local/ldb-toolchain/include/c++/11/bits/std_thread.h:211
17# execute_native_thread_routine at ../../../../../libstdc++-v3/src/c++11/thread.cc:84
18# start_thread in /lib64/libpthread.so.0
19# __clone in /lib64/libc.so.6

**Comment by LemonLiTree** [comment]
做过什么其他操作吗？
